src/base/types.h

```diff
@@ -35,7 +35,7 @@ const qlonglong MAX_ETA = 8640000;
 
 enum class ShutdownAction
 {
-    None,
+    Exit,
```

Comment:
As to me, the suggested name change is confusing. Take the `Suspend` for example. Does it mean that the app will exit and the system be suspended _in addition_  to that or the system will be _just_ suspended? The old identifiers were saying that the system will be suspended in addition to the exit.